ja/lessons/basics/enum.md

```diff
@@ -167,9 +166,9 @@ iex> Enum.reduce(["a","b","c"], "1", fn(x,acc)-> x <> acc end)
 
 ### sort
 
-コレクションをソートするのは1つではなく、2つあるソート関数を使えば簡単です。
+コレクションのソートに、関数を1つだけでなく2つ使うと容易になります。
 
-`sort/1`はErlangのterm orderingを使ってソート順序を決めるというものです:
+`sort/1`はソートの順序を決める為にErlangの型順序を使います:
```

Comment:
Erlang term is a part of types but, not all types. 😢

Reply:
I intend to say that.
What is the proper Japanese translation of `Term Ordering`?

Reply:
How about `ターム優先順位` or `Term 優先順位`?

Reply:
Because When I heard this word, I thought it was similar to the [operator precedence](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/pages/Operators.md#operator-precedence-and-associativity), except that it was about the term.

I think it 's better to leave the term rather than translate it into Japanese. There is no proper Japanese word in [this definition](http://erlang.org/documentation/doc-6.0/doc/reference_manual/data_types.html).

> Erlang provides a number of data types which are listed in this chapter. A piece of data of any data type is called a term.

Reply:
Oh, you can say that again 👍 
I have realised we don't have to bother to translate that.
Therefore, I will use `Term Ordering` as it is intact and push it newly.
As you say, my translation can be misleading.

Thank you!

Reply:
I'm sorry to be late, but I have pushed it newly.
Please review.

Reply:
umm... Did you read other comments?

Reply:
Oh... 😭 
I believed I replaced the word with `Term 優先順位` of your ideas.
Please wait for a moment.

Reply:
How is this?

Reply:
No no, `Term Ordering` is good enough to me. I means those comments:
https://github.com/elixirschool/elixirschool/pull/1905#discussion_r300895610
https://github.com/elixirschool/elixirschool/pull/1905#discussion_r300895493
https://github.com/elixirschool/elixirschool/pull/1905#discussion_r300894987

Reply:
Sorry, I've noticed them now...

Reply:
😅